src/html.jl

```diff
@@ -206,3 +206,32 @@ function notebook2html(path::AbstractString)
     return notebook2html(notebook)
 end
 
+"""
+NOTE: NEW API for SessionActions support
```

Comment:
Please make this a valid docstring with the method name on the first line and a description for the user of what it does.

For example, see any Julia source file like https://github.com/JuliaLang/julia/blob/master/stdlib/Dates/src/parse.jl.